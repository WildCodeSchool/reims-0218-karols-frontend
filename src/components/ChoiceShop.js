import React from "react"
import {
  Card,
  CardTitle,
  CardText,
  CardImgOverlay,
  CardFooter,
  Button
} from "reactstrap"

const ChoiceShop = ({ city, image, adresse }) => {
  return (
    <div className="mb-4">
      <Card inverse>
        <img
          className="img-fluid"
          width="500"
          height="250"
          src={image}
          alt="Name of service"
        />

        <CardImgOverlay>
          <CardTitle
            style={{
              fontSize: "2em"
            }}
          >
            {city}
          </CardTitle>
          <CardText>{adresse}</CardText>
        </CardImgOverlay>
        <CardFooter>
          <Button color="default" size="lg" block>
            Choisir ce service
          </Button>
        </CardFooter>
      </Card>
    </div>
  )
}

export default ChoiceShop
