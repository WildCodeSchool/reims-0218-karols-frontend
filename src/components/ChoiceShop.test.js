import React from "react"
import ReactDOM from "react-dom"
import ChoiceShop from "./ChoiceShop"
import renderer from "react-test-renderer"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<ChoiceShop />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer.create(<ChoiceShop />).toJSON()
  expect(tree).toMatchSnapshot()
})
