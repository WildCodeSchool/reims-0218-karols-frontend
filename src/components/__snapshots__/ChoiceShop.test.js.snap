// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="mb-4"
>
  <div
    className="card text-white"
  >
    <img
      alt="Name of service"
      className="img-fluid"
      height="250"
      src={undefined}
      width="500"
    />
    <div
      className="card-img-overlay"
    >
      <h5
        className="card-title"
        style={
          Object {
            "fontSize": "2em",
          }
        }
      />
      <p
        className="card-text"
      />
    </div>
    <div
      className="card-footer"
    >
      <button
        className="btn btn-default btn-lg btn-block"
        onClick={[Function]}
        type={undefined}
      >
        Choisir ce service
      </button>
    </div>
  </div>
</div>
`;
